projects/models.py

```diff
@@ -9,12 +9,18 @@ class Project(models.Model):
     title = models.CharField(max_length=200)
     description = models.TextField()
     max_members = models.IntegerField()
-    difficulty = ChoiceField(choices=['beginner', 'intermediate', 'advanced'])
+    difficulty_choices = (
+        ('beginner', 'beginner'),
+        ('intermediate', 'intermediate'),
+        ('advanced', 'advanced')
+        )
+    difficulty = models.CharField(max_length=15, choices=difficulty_choices)
     # Project, user attributes
     owner = models.ForeignKey('auth.User', related_name='owns_project')
     participants = models.ManyToManyField('auth.User', related_name='participant_of_projects',
                                           blank=True)
     coach = models.ForeignKey('auth.User', related_name='coach_of_projects', null=True, blank=True)
+    created_date = models.DateTimeField(default=timezone.now)
 
```

Comment:
I think `created_date` is misleading as it is not a date it is a datetime, maybe `created_at` is more accurate. We will need to take a look also of if timezone.now or timezone.utcnow is the correct thing to do